packages/editor/src/components/post-permalink/editor.js

```diff
@@ -49,7 +49,7 @@ class PostPermalinkEditor extends Component {
 				className="editor-post-permalink-editor"
 				onSubmit={ this.onSavePermalink }
 			>
-				<span>
+				<span className="editor-post-permalink-editor__container">
```

Comment:
It's close, but this is not quite following the [naming standards](https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#naming). Specifically, the part preceding the `__` isn't concerned at all with the file name, only the top-level folder and the immediate parent folder. It should be `editor-post-permalink__editor-container`.

Reply:
Looking again at those standards, we need to update them to be less... editor-specific.